my_game.py

```diff
@@ -482,7 +484,7 @@ def on_show_view(self):
 
         # Spawn Asteroids
         for r in range(ASTEROIDS_PR_LEVEL):
-            self.asteroid_list.append(Asteroid())
+            self.asteroid_list.append(Asteroid(random.randrange(1, 4)))
```

Comment:
Please make the Asteroid choose a valid random size, if no size is requested (we should not request a specific size here).